examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -21,6 +21,8 @@
         "Test_TC_APPLAUNCHER_3_7",
         "Test_TC_APPLAUNCHER_3_8",
         "Test_TC_APPLAUNCHER_3_9",
+        "TC_MEDIAPLAYBACK_6_8",
```

Comment:
This is really a large number of tests that should be reviewed by a different set of people.Can we please start breaking these PRs up and getting them properly reviewed by the appropriate feature devs if they're not going to write their own scripts? The PR title really doesn't indicate to any of them that they should be reviewing.

@raju-apple, @akrithika27